testCourse/questions/truthTableDemo/README.md

```diff
@@ -0,0 +1,45 @@
+# Training: Truth Table
+
```

Comment:
Documentation should go in `docs/elements.md` and should follow the same style as the documentation for the other elements there.